examples/RunTemporaryCommand/runTempCommand.go

```diff
@@ -0,0 +1,291 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/TheJumpCloud/jcapi"
+)
+
+const (
+	COMMAND_NAME_RANDOM_PART int = 16
+
+	MAX_COMMAND_LEN int = 16384 // Maximum size of a command to send to JumpCloud
+
+	RESULT_POLL_TIME     int = 5  // Check every RESULT_POLL_TIME seconds for command results
+	RESULT_MAX_POLL_TIME int = 70 // Stop checking for results after RESULT_MAX_POLL_TIME_SECONDS (must be a minimum of 60 seconds)
+
+	URL_BASE string = "https://console.jumpcloud.com/api"
+)
+
+func makeImmediateCommand(name, command, commandType, shell, user string) (cmd jcapi.JCCommand) {
+	cmd = jcapi.JCCommand{
+		Name:        name,
+		Command:     command,
+		CommandType: commandType,
+		User:        user,
+		LaunchType:  "manual",
+		Schedule:    "immediate",
+		Timeout:     "0", // No timeout
+		ListensTo:   "",
+		Trigger:     "",
+		Sudo:        false,
+		Shell:       shell,
+		Skip:        0,
+		Limit:       10,
+	}
+
+	return
+}
+
+func deleteCommandResultsByName(jc jcapi.JCAPI, commandName string) (err error) {
+	results, err := jc.GetCommandResultsByName(commandName)
+	if err != nil {
+		err = fmt.Errorf("Could not find the command result for '%s', err='%s'", commandName, err.Error())
+		return
+	}
+
+	errBuf := ""
+
+	for _, result := range results {
+
+		err = jc.DeleteCommandResult(result.Id)
+		if err != nil {
+			errBuf += err.Error() + "\n"
+			err = nil
+		}
+	}
+
+	if errBuf != "" {
+		err = fmt.Errorf("One or more deletes failed, err='%s'", errBuf)
+		return
+	}
```

Comment:
Small detail, but this puts an extra "\n" at the end of the output, perhaps intentionally?  One could probably improve readability and flexibility (and memory allocations) by declaring `errors := make([]string, 0, len(results))` and then append to buffer and using `strings.join` at the end.